### PR TITLE
Remove python 2.6 and 3.3 from contributing doc

### PR DESCRIPTION
**Pull request by larrybradley**
and a minor grammar correction.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ in the package.
 **Code Quality**
   * Are the [coding guidelines](http://docs.astropy.org/en/latest/development/codeguide.html)
     followed?
-  * Is the code compatible with Python 2.6, 2.7, as well as >=3.3?
+  * Is the code compatible with Python 2.7 and >=3.3?
```
Comment (pllim) [inline CONTRIBUTING.md]: Didn't we also remove some 3.3 stuff? In other words, did we bump Python 3 minversion to beyond 3.3? Maybe @bsipocz remembers.

Comment (bsipocz) [inline CONTRIBUTING.md]: Good catch! Definitely remove python 3.3, too, it's not supported in 2.0.

Comment (MSeifert04) [inline CONTRIBUTING.md]: APE-10 says we support Python 3.4+

Comment (larrybradley) [inline CONTRIBUTING.md]: Done, thanks!

**Comment by bsipocz** [comment]
Thanks @larrybradley!

**Pull request merged**